web/themes/joinup/prototype/sass/components/_search-bar.sass

```diff
@@ -70,7 +70,6 @@
     width: 200px
   .search-bar__input
     font-weight: $font-weight-thin
-    padding: 7px 0
```

Comment:
Maybe we could move this issue to ticket ISAICP-3619 which is related with header search behaviour and displaying?